src/main/resources/electronic-data-mapping.json

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
Should this file be renamed to something more general than "electronic-data-mapping" because it contains more than just electronic data mapping? Is that the idea? if so, I had "data-mapping" in my branch, or we could rename it to "order-data-mapping" or something like that?

Reply:
We should have one combined file. Once we merge both of these PRs (I saw the data-mapping.json file in your branch and didn't want to collide) we'll need to combine the mappings into one file. The final file name doesn't matter apart from understandability, but "data-mapping.json" is fine with me. :)